src/pages/notes/david-copperfield/index.mdx

```diff
@@ -19,22 +19,108 @@ import HowToCite from "/src/components/HowToCite.astro";
   img="/images/david_copperfield/DCIV.jpg"
 />
 
+{/* prettier-ignore */}
+<TOC>
+  - [_Copperfield_ in Context](#copperfield-in-context)
+  - [Titling the Novel: _Copperfield_'s Genre and Form](#titling-the-novel-copperfields-genre-and-form)
```

Comment:
Noting that in the deploy preview, the apostrophe runs into the italicized "d." I'm not sure whether this is an issue with it being italics, our font, or the combination thereof. You might try including the apostrophe inside the italics just in case the visual is better, or just not worrying about it if you don't think it's an issue.